extensions/patterns/configure_ec2/README.md

```diff
@@ -4,6 +4,23 @@
 
 This pattern is designed to help get an EC2 instance up and running.
 
+To enable SSH access to the EC2 instance from your local machine, you need to do 2 things:
+
+1. **Provide a Key Name**: Enter a key name in the **key_name** parameter in the survey. A new key will be created (or an existing key with the specified namewill be used) and associated with the EC2 instance. If creating a new key pair, be sure to save the private key value provided at the end of the job run, as you'll need it for future access to the instance.
+
+2. **Add a Security Group Rule for SSH Access**: Configure a security group rule to allow inbound SSH traffic from your local machine's IP address. Provide this rule in the **sg_rules** parameter in the survey.
+
+_Sample Security Group Rule_
+---------------------------
+
+```yaml
+sg_rules:
```

Comment:
```suggestion
```
I would remove the `sg_rules` part of this since it doesn't get included in the survey field input.